brands/shop/tyres.json

```diff
@@ -66,6 +66,17 @@
       "shop": "tyres"
     }
   },
+  "shop/tyres|Eurotyre~(Portugal)": {
+    "countryCodes": ["pt"],
+    "matchTags": ["shop/car_repair"],
+    "tags": {
+      "brand": "Eurotyre",
+      "brand:wikidata": "Q3060871",
```

Comment:
This wikidata item is for [Eurotyre (France) (Q3060871)](https://www.wikidata.org/wiki/Q3060871) 😉